src/pydicom/pixels/processing.py

```diff
@@ -297,6 +297,82 @@ def apply_modality_lut(arr: "np.ndarray", ds: "Dataset") -> "np.ndarray":
     return arr
 
 
+def apply_presentation_lut(arr: "np.ndarray", ds: "Dataset") -> "np.ndarray":
+    """Apply a Presentation LUT to `arr` and return the P-values.
+
+    Parameters
+    ----------
+    arr : numpy.ndarray
+        The :class:`~numpy.ndarray` to apply the presentation LUT operation to.
+    ds : dataset.Dataset
+        A dataset containing :dcm:`Presentation LUT Module
+        <part03/sect_C.11.4.html>` elements.
+
+    Returns
+    -------
+    numpy.ndarray
+        If a Presentation LUT Module is present in `ds` then returns an array
+        of P-values, otherwise returns `arr` unchanged.
+
+    Notes
+    -----
+    If `ds` contains a Modality LUT and/or VOI LUT then they must be applied
+    before the Presentation LUT.
+
+    See Also
+    --------
+    :func:`~pydicom.pixels.processing.apply_modality_lut`
+    :func:`~pydicom.pixels.processing.apply_voi_lut`
+    """
+    if "PresentationLUTSequence" in ds:
```

Comment:
Presentation LUT sequences only appear in presentation states, not in the images they are applied to (other than presentation LUT shape, which may appear in both). 
To make this useful, you would need a source ds (the presentation state) and a destination ds. Reading the destination from the PS is probably not helpful, as you would have to find the image with the correct SOPInstanceUID.

Reply:
Ah!

This is where having experience with these things would help.

Reply:
Although doesn't `arr` come from the destination ds, anyway?
```python
arr = src_ds.pixel_array
# apply modality/VOI etc
pvalues = apply_presentation_lut(arr, presentation_ds)
```
Is the dataset with the pixel data needed by the time you apply the Presentation LUT? I'd still need to reword the note, though.

Reply:
Ah, you are right - I have misunderstood this. So you can mostly disregard my comment!